desktop/apps/consignments2/client/index.js

```diff
@@ -0,0 +1,18 @@
+import { data as sd } from 'sharify'
+import Artworks from '../../../collections/artworks.coffee'
+import ArtworkRailView from '../../../components/artwork_rail/client/view.coffee'
+import multiPageView from '../../../components/multi_page/index.coffee'
+
+module.exports = ->
```

Comment:
Might be nicer to use `export default` (which is used elsewhere as `require('..').default` to keep in ES20XX land.